aten/src/ATen/Declarations.cwrap

```diff
@@ -3193,7 +3193,8 @@
     - THTensor* tensor2
 ]]
 [[
-  name: gesv
+  name: _gesv
```

Comment:
I think we can be clearer with the name here. `_gesv_nonbatch` or `_gesv_single` is probably better.